src/Test/All.hs

```diff
@@ -8,6 +9,10 @@ import Data.List
 import System.Directory
 import System.FilePath
 
+#if __GLASGOW_HASKELL__ < 710
+import Data.Functor ((<$>))
+#endif
```

Comment:
Instead of this just `import Data.Functor` and on the next line do `import Prelude`. No CPP and it's warning free on all GHC versions.